Source/Core/DiscIO/DirectoryBlob.cpp

```diff
@@ -471,7 +470,21 @@ DirectoryBlobReader::DirectoryBlobReader(
       }
     }
 
-    SetPartitions(std::move(partitions));
+    SetPartitions(std::move(partitions), std::move(volume));
```

Comment:
You can't use `volume` here. It's already been moved from.

Actually, is `SetPartitions` using its volume parameter for anything at all? Could it be removed?